modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a subscription
+
+You can create a subscription to connect a service or other event sink to a channel.
+
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* If you do not have an existing sink that you wish to use, you can create a Service to use as a sink by following the documentation on xref:../../serverless/serving-creating-managing-apps.adoc#serving-creating-managing-apps[Creating and managing serverless applications].
+* You will need a channel to connect your subscription to. See xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using channels with Knative Eventing].
+
+.Procedure
+
+. Create a Subscription object to connect a channel to a service, by creating a YAML file containing the following:
++
+[source,yml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Subscription
+metadata:
+  name: my-subscription <1>
+  namespace: default
+spec:
+  channel: <2>
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: Channel
+    name: example-channel
+  delivery: <3>
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: error-handler
+  subscriber: <4>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
++
+<1> Name of the subscription.
+<2> Configuration settings for the channel that the subscription connects to.
++
+[NOTE]
+====
```

Comment:
This Note is throwing off the callout numbering. After some research/testing, this is a bit tricky due to the callouts being within a procedure and the note being within the callouts :sweat_smile:. I was able to get the intended outcome by doing this:
```
<last line of yaml code>
----
+
--
<1> text
<2> text
+
[NOTE]
====
text
====
<3> text
<4> text
--
```
There may be a better way, but I couldn't find it.

Reply:
I actually think it's OK to remove this because it's mentioned earlier, my bad. Thanks though!